src/rules/Rule_4_10_2.cpp

```diff
@@ -0,0 +1,46 @@
+//===-  Rule_4_10_2.cpp - Checker for MISRA C++ 2008 rule 4-10-2-------------===//
```

Comment:
So you have a defined max line length? 
I've run clang-format on Rule_4_10_2.cpp and it actually has broken this line but I've reverted it as in other sources the line is actually that long.